internal/engine/wazevo/backend/isa/arm64/instr.go

```diff
@@ -890,9 +908,21 @@ func (i *instruction) String() (str string) {
 	case vecMiscNarrow:
 		panic("TODO")
 	case vecRRR:
-		panic("TODO")
+		str = fmt.Sprintf("%s %s.%s, %s.%s",
+			vecOp(i.u1),
+			formatVRegSized(i.rd.nr(), 64),
+			vecArrangement(i.u2),
+			formatVRegSized(i.rn.nr(), 64),
+			vecArrangement(i.u2),
+		)
 	case vecMisc:
-		panic("TODO")
+		str = fmt.Sprintf("%s %s.%s, %s.%s",
+			vecOp(i.u1),
+			formatVRegSized(i.rd.nr(), 64),
+			vecArrangement(i.u2),
+			formatVRegSized(i.rn.nr(), 64),
+			vecArrangement(i.u2),
+		)
```

Comment:
the 64-bit size has the same potential flaw as the change above

Reply:
we can fix it later, once we are at the phase when implementing SIMD spec. for now let's just not get concerned that much

Reply:
vector operations need arrangement to correctly print. Could you add `formatVRegSizedVec` or something like that which accepts `vecArrangement` as well? While there, could you add all the arrangement needed in the future? you can check the existing compiler's code which arrangement exists